frame/treasury/src/lib.rs

```diff
@@ -476,6 +549,64 @@ pub mod pallet {
 			Ok(())
 		}
 
+		/// Propose and approve a spend of treasury funds.
+		///
+		/// - `origin`: Must be `SpendOrigin` with the `Success` value being at least `amount`.
+		/// - `amount`: The amount to be transferred from the treasury to the `beneficiary`.
+		/// - `beneficiary`: The destination account for the transfer.
+		///
+		/// NOTE: For record-keeping purposes, the proposer is deemed to be equivalent to the
+		/// beneficiary.
+		#[pallet::call_index(5)]
+		#[pallet::weight(T::WeightInfo::spend())]
+		pub fn spend(
+			origin: OriginFor<T>,
+			asset_id: T::AssetKind,
```

Comment:
How do I spend a native tokens? on a remote chain.
native token does not have an id

Reply:
The assetKind can be set to `()`

Reply:
But if I remember correctly, the native token should still have an assetID on the remote chain. Since it's just one more asset in statemint. This needs some testing on my part.

Reply:
probably worth renaming the `asset_id` to `asset_kind` as well. 
most probably we will have a enum here, Native, and AssetId(u32).
no, there is no id for native currency, on Statemint as well.